src/lib/DiscordWorker.ts

```diff
@@ -102,10 +102,10 @@ function addCustomMethods(job: DiscordJob) {
   }
 
   job.setThreadName = async (name) => {
```

Comment:
this seems like a workaround that isn't needed if we just do something like this in the precheck, if i understand correctly:

```
      if(job.data.threadId) {
        await job.setThreadName(companyName)
      }
```

i dont see a reason to call the setThreadName if there is no thread